Sources/Clappr_iOS/Classes/Animation/QuickSeekAnimation.swift

```diff
@@ -1,5 +1,5 @@
 class QuickSeekAnimation {
-    private var core: Core?
+    private weak var core: Core?
```

Comment:
I'm analyzing the code and found the use of Core is to provide a view that receives animation from the `QuickSeekPlugin`. Is there another way to avoid this bad behavior (retain cycle on this object)? 
One suggestion maybe is split the responsibility between Plugin and the View, and the View do not know the Core. What do you think?

Reply:
Yes. We actually don't need to store the reference to the core nor the view here, we can just pass it through.

Reply:
I'm thinking about the same approach. Make more sense

Reply:
It's done! o/